Translators/WZDx/snmp_operations.py

```diff
@@ -0,0 +1,19 @@
+import subprocess
+import logging
+
+def clear_index(rsu):
```

Comment:
There's a [TimDeleleteController](https://github.com/usdot-jpo-ode/jpo-ode/blob/6a497f2710a7cbeb23770b5df139180ee56e25bc/jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/traveler/TimDeleteController.java#L62) in the jpo-ode that we could call into instead of using this manual instance. Thoughts? Future iteration maybe

Reply:
I think that would make a lot of sense and prevent code duplication - my only thought would be if using that method would be as performant b/c we'd need to send a number of requests through to the ODE to delete messages.

Reply:
We've used it previously, it's doing much the same as we are doing here. The only real difference is that we'd be doing a REST call on top of it. I think the overhead is probably worth the gain in code de-duplication.

Reply:
Updated this to use the TimDeleteController instead of running snmp commands manually